playbooks/openshift-apps/fedora-coreos-koji-tagger.yml

```diff
@@ -0,0 +1,55 @@
+- name: make the app be real
+  hosts: os_masters_stg[0]
+  user: root
+  gather_facts: False
+
+  vars_files:
+    - /srv/web/infra/ansible/vars/global.yml
+    - "/srv/private/ansible/vars.yml"
+    - /srv/web/infra/ansible/vars/{{ ansible_distribution }}.yml
+
+  roles:
+  - role: openshift/project
+    app: fedora-coreos-koji-tagger
+    description: Tags builds into koji tags within Fedora's Koji instance
+    appowners:
+    - dustymabe
+    - mizdebsk
+    - jlebon
+
+# I need either this keytab role or to define the keytab in the
+# secret below not sure which
+  - role: openshift/keytab
+    app: fedora-coreos-koji-tagger
+    key: koji-keytab
+    secret_name: fedora-coreos-koji-tagger-coreos-bot-keytab
+    service: bodhi
```

Comment:
You can use `service: coreos-koji-tagger`

Reply:
+1 - done